Cython/Compiler/Symtab.py

```diff
@@ -1175,6 +1177,8 @@ def __init__(self, name, parent_module, context):
         self.module_name = EncodedString(self.module_name)
         self.context = context
         self.module_cname = Naming.module_cname
+        self.modulestate_cname = Naming.modulestate_cname
+        self.modulestateglobal_cname = Naming.modulestateglobal_cname
```

Comment:
Hmmm … Is `self.modulestate_cname` ever something else than `Naming.modulestate_cname` ? Because if not, then this is a useless attribute.

Reply:
Yup, you were right. Removed.